.github/workflows/pr-build.yml

```diff
@@ -41,14 +41,14 @@ jobs:
       # Set fetch-depth: 0 to fetch commit history and tags for use in version calculation
     steps:
       - name: Check out code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           ref: ${{github.event.pull_request.head.ref}}
           repository: ${{github.event.pull_request.head.repo.full_name}}
           fetch-depth: 0
 
       - name: Setup buf
-        uses: bufbuild/buf-setup-action@v1.9.0
+        uses: bufbuild/buf-setup-action@v1
```

Comment:
Is this a downgrade?

Reply:
No - I just unpinned the minor/patch version (leaving major version pinned) to keep it consistent with the next two steps. The latest is 1.31.0 so this is effectively an upgrade.

Reply:
I thought so. Just wanted to make sure.